docs/deploy/pg_lakehouse.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: pg_lakehouse
+---
+
+This guide explains how to install the `pg_lakehouse` extension inside an existing PostgreSQL database. These
+instructions are for deploying on Linux Debian/Ubuntu. If you are using a different version of Linux, please
+[contact us](mailto:sales@paradedb.com).
+
+# Prerequisites
+
+First, ensure that you have superuser access to the Postgres database.
+
+Next, install `libicu`.
+
+```bash
+sudo apt-get install -y libicu70
+```
+
+# Install `pg_lakehouse`
+
+ParadeDB provides prebuilt binaries for the `pg_lakehouse` extension for Postgres 15 and 16 on Ubuntu 22.04. They can be found in [GitHub Releases](https://github.com/paradedb/paradedb/releases/latest). If you
+are using a different version of Postgres or a different operating system, you will need to build the extension from source.
+
+## Using Prebuilt Binaries
+
+```bash
+curl -L "https://github.com/paradedb/paradedb/releases/download/v0.6.1/pg_lakehouse-v0.6.1-pg16-arm64-ubuntu2204.deb" -o /tmp/pg_lakehouse.deb
```

Comment:
I suggest we do bump to v0.7.0 once we release `pg_lakehouse`, and since that will be the first release we should mention it here cuz there is no pg_lakehouse v0.6.1